runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -90,6 +90,9 @@ public T open() {
 
         if (streamName != null) {
             streamID = CorfuRuntime.getStreamID(streamName);
+            log.info("CorfuObject: creating stream {} id {}", streamName, streamID.toString());
```

Comment:
I think you can consolidate the log lines, printing null is fine.

Reply:
done.